dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -146,6 +132,38 @@ def run(args, write_profile=None):
         t1 = clock()
         wait(ddf_join.persist())
         took = clock() - t1
+    return took
+
+
+def merge_explicit_comms(args, ddf1, ddf2):
+    t1 = clock()
+    wait(explicit_comms.cudf_merge(ddf1, ddf2).persist())
+    took = clock() - t1
+    return took
+
+
+def run(client, args, write_profile=None):
+    # Generate random Dask dataframes
+    ddf_base = get_random_ddf(
+        args.chunk_size, args.n_workers, args.frac_match, "build", args
```

Comment:
Just a note that I started experimenting with a simple [`set_index` benchmark](https://github.com/rjzamora/dask-cuda/blob/set_index_bench/dask_cuda/benchmarks/local_cudf_set_index.py) (mostly a copy of the current local_cudf_merge.py).  I noticed that the underlying `sort_index` method in cudf was the bottleneck for a 1-to-1 mapping between workers (devices) and partitions.  I added the option to create multiple partitions per worker (`args.chunks_per_worker`) and noticed a dramatic drop in performance with `args.chunks_per_worker>1` for the same total dataframe size.  In this case communication, concatenation and `group_split_cudf` become much more time-intensive.

Perhaps it makes sense to add the same argument here (to capture a more-realistic dask use-case)?